care/facility/models/patient.py

```diff
@@ -571,17 +571,16 @@ def format_diagnoses(diagnosis_ids):
 class PatientMetaInfo(models.Model):
     class OccupationEnum(enum.Enum):
         STUDENT = 1
-        MEDICAL_WORKER = 2
-        GOVT_EMPLOYEE = 3
-        PRIVATE_EMPLOYEE = 4
-        HOME_MAKER = 5
-        WORKING_ABROAD = 6
-        OTHERS = 7
```

Comment:
we should keep the existing choices

Reply:
@sainak but In the issue, only a few options for occupation were specified https://github.com/coronasafe/care_fe/issues/6945#issue-2059884492

Reply:
https://github.com/coronasafe/care_fe/issues/6945#issue-2059884492

@sainak In the issue, only a few options for occupation were specified

Reply:
@konavivekramakrishna that'd cause old data to be mixed with new data. A medical worker patient before would end up becoming a businessman after this.

@sainak but we were not recording these data so far (only 22 records are there in staging which was captured years ago). So if we just clean the staging, we could have a cleaner migration right?

Reply:
yes we can after confirmation from prod

Reply:
@rithviknishad @sainak 

can you please conform regarding the occupation choices

Reply:
should I add new choices to existing choices or remove them

Reply:
@sainak we are safe to delete.

@Ashesh3 had checked the prod. instances

cc: @konavivekramakrishna